s3/src/main/scala/geotrellis/spark/io/s3/MultibandGeoTiffS3InputFormat.scala

```diff
@@ -8,10 +8,10 @@ import org.apache.hadoop.mapreduce.{InputSplit, TaskAttemptContext}
 /** Read multi band GeoTiff from S3 */
 class MultibandGeoTiffS3InputFormat extends S3InputFormat[ProjectedExtent, MultibandTile] {
```

Comment:
Seems like these InputFormats are no longer used, and we're just reading in the bytes. Should we delete?